common/bin-common/src/logging/mod.rs

```diff
@@ -24,6 +28,18 @@ pub fn setup_logging() {
         .init();
 }
 
+pub fn setup_tracing(file_name: &str) {
```

Comment:
if we're using a global `setup_tracing`, should it always log to a file? It sounds like something we'd want to make configurable